_includes/0.3/left_sidebar.html

```diff
@@ -58,6 +58,7 @@
         <a href="{% link docs/0.3/grid_product_smart_contract_specification.md %}">Product Specification</a>
```

Comment:
There should also be a link to the feature document here https://grid.hyperledger.org/community/planning/roadmap.html

Reply:
Thanks, updated.